pom.xml

```diff
@@ -94,10 +94,9 @@
   </distributionManagement>
 
   <properties>
-    <java.version>6</java.version>
+    <java.version>8</java.version>
     <maven.compiler.source>${java.version}</maven.compiler.source>
     <maven.compiler.target>${java.version}</maven.compiler.target>
-    <maven.compiler.release>${java.version}</maven.compiler.release>
```

Comment:
This was there because Thymeleaf is being currently compiled with JDK 11. even if tests are passed also with previous versions of the JDK, compilation is done with 11, not 8, simply out of an understanding that as long as bytecode and runtime compatibility are kept (hence the `maven.compiler.release`), using the more modern (and now almost _de facto_ standard) version of the compiler could bring some optimization profit.

Do you see any issues with keeping this JDK 11-based compilations?

Reply:
Sorry this was caused by me using the wrong JDK to build. I have reverted this.